dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/IEC61400/BaseClasses/BaseWPPQControl.mo

```diff
@@ -49,7 +48,7 @@ model BaseWPPQControl "Reactive power control base module for wind power plants
 
   Modelica.Blocks.Math.Feedback feedback annotation(
     Placement(visible = true, transformation(origin = {-220, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Modelica.Blocks.Tables.CombiTable1Ds combiTable1Ds2(extrapolation = Modelica.Blocks.Types.Extrapolation.HoldLastPoint, table = TableQwpUErr) annotation(
+  Modelica.Blocks.Tables.CombiTable1Ds combiTable1Ds2(extrapolation = Modelica.Blocks.Types.Extrapolation.HoldLastPoint)  annotation(
```

Comment:
Why remove the table's name?  We have the same one for 2015 and 2020 no?

Reply:
Considering that the tables are not declared in this base model BaseWPPQControl.mo (because they are different between 2015 and 2020 versions) I don't think it is possible to call them here. So I've done it WPPQControl2015 and WPPQControl2020